### PR TITLE
Feature/custom personalization widget

### PR DESCRIPTION
**Pull request by Janee**
:question: What
---
I updated this repo to work with the widget branding and customization feature.

:speech_balloon: Why
---
Belvo's customers need to know this feature is available.

### CODE REVIEW

File: ruby/app.rb [hunk 1]
```diff
@@ -33,6 +40,7 @@
 options "*" do
   response.headers["Allow"] = "HEAD,GET,PUT,POST,DELETE,OPTIONS"
   response.headers["Access-Control-Allow-Headers"] = "X-Requested-With, X-HTTP-Method-Override, Content-Type, Cache-Control, Accept"
+  response.headers['Access-Control-Allow-Origin'] = '*'
```
Comment (shekodn) [inline ruby/app.rb:43]: do we need to allow everything?

**Review by AlbertMaristany — APPROVED** [review]
Looks good to me. Gonna page @jrubendiaz to get a closer look :eyes:

**Review by jrubendiaz — APPROVED** [review]
LGTM too 👍

File: ruby/app.rb (hunk 1 above)
Comment (Janee) [reply ruby/app.rb:43]: According to Mozilla, they recommend leaving the `*` value if you will use public APIs (like us).
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin

**Review by shekodn — APPROVED** [review]

**Pull request merged**